frontend/src/components/auth/components/home.tsx

```diff
@@ -0,0 +1,5 @@
+import { FC } from 'react';
```

Comment:
this page in the auth scope. we need to avoid it

move it to the frontend/src/components/dashboard/dashboard.tsx